Resources/views/frontend/_public/src/js/jquery.swag-paypal-unified-sepa.js

```diff
@@ -194,7 +194,10 @@
                 theme: 'light'
             });
 
-            actions.redirect($.swagPayPalRenderUrl(this.opts.returnUrl, params));
+            var url = $.swagPayPalRenderUrl(this.opts.returnUrl, params);
```

Comment:
below you added the URL to the first var declaration. would be nice, if you could make it consistent